package.json

```diff
@@ -22,22 +23,31 @@
     "discord.rss": "5.0.3",
     "dotenv": "8.2.0",
     "erlpack": "github:discordapp/erlpack",
+    "lodash": "^4.17.15",
     "node-fetch": "2.6.0",
+    "nyc": "^15.0.0",
     "polish-plurals": "1.1.0",
+    "typescript": "^3.7.4",
+    "vm2": "^3.8.4",
```

Comment:
This library is unsafe and we won't use it. It allows one to escape the sandbox quite easily which makes it completely useless.
https://github.com/patriksimek/vm2/issues/180

Reply:
Possible alternative: https://github.com/laverdet/isolated-vm

Reply:
Ok, I will switch to that, but it will require more time.